Kickstarter-iOS/Views/Cells/CommentRemovedCell.swift

```diff
@@ -99,27 +130,64 @@ private let rowStackViewStyle: StackViewStyle = { stackView in
 
 // MARK: - Functions
 
-// TODO/FIXME: Allow "Learn more about comment guidelines." to be tappable and open link in a web browser
-
 private func attributedTextCommentRemoved() -> NSAttributedString {
-  let regularFontAttribute = [
-    NSAttributedString.Key.font: UIFont.ksr_callout(),
-    NSAttributedString.Key.foregroundColor: UIColor.ksr_support_400
+  let regularFontAttribute: String.Attributes = [
+    .font: UIFont.ksr_callout(),
+    .foregroundColor: UIColor.ksr_support_400
   ]
-  let coloredFontAttribute = [
-    NSAttributedString.Key.font: UIFont.ksr_callout(),
-    NSAttributedString.Key.foregroundColor: UIColor.ksr_create_700
+  let coloredFontAttribute: String.Attributes = [
+    .font: UIFont.ksr_callout(),
+    .foregroundColor: UIColor.ksr_create_700,
+    .underlineStyle: 0
   ]
 
-  let attributedString = NSMutableAttributedString(
+  let removedCommentAttributedString = NSMutableAttributedString(
     string: Strings.This_comment_has_been_removed_by_Kickstarter(),
     attributes: regularFontAttribute
   )
-  let learnMoreAttributedString = NSMutableAttributedString(
-    string: Strings.Learn_more_about_comment_guidelines(),
-    attributes: coloredFontAttribute
-  )
-  attributedString.append(learnMoreAttributedString)
 
-  return attributedString
+  guard let communityGuidelinesLink = HelpType.community
```

Comment:
Wondering if it makes sense to hardcode `community` in two places or if the delegate should pass back the `HelpType` 🤔

Reply:
Good catch. Removing this line here in favor of the `mapConst` in `CommentsViewModel`.

Reply:
I guess what I'm saying is that maybe we should just pass the URL through? The hard-coding prevents us from easily using a different URL in the tappable text or maybe more than one tappable link for example.

Reply:
We do this here: https://github.com/kickstarter/ios-oss/blob/master/Library/ViewModels/PledgeViewCTAContainerViewModel.swift#L44

Maybe we should make that a static func on `HelpType` for reusability.

Reply:
Updated!